orchestrator-salt/src/main/resources/salt/salt/postgresql/scripts/backup_db.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ $# -ne 6 ]; then
+  echo "Invalid inputs provided"
+  echo "Script accepts five inputs:"
+  echo "  1. Cloud Provider (azure | aws)"
+  echo "  2. PostgreSQL host name."
+  echo "  3. PostgreSQL port."
+  echo "  4. Object Storage Service url to place backups."
+  echo "  5. PostgreSQL user name."
+  echo "  6. PostgreSQL password."
```

Comment:
Re-order the inputs to be in-order. Move Object store URL to the end so that all the database configurations will be together.

Reply:
My intent with password last was to make it easier to remove the password altogether if/when we migrate to using the recommended `.pgpass` file for passwords.

How about
```suggestion
  echo "  1. Cloud Provider (azure | aws)"
  echo "  2. Object Storage Service url to place backups."
  echo "  3. PostgreSQL host name."
  echo "  4. PostgreSQL port."
  echo "  5. PostgreSQL user name."
  echo "  6. PostgreSQL password."
```

Reply:
Make sure you don't actually delete the part that explains they're missing parameters. And you might explicitly mark `PostgresSQL password` as `optional`. I didn't realize it was, which is why I ticked the mentioned number of parameters.

Reply:
Ah, you're giving me too much credit, I hadn't thought of making it optional. I'll add the accoutrements to make it so.